addons/website/views/website_templates.xml

```diff
@@ -285,10 +285,30 @@
         </t>
     </xpath>
 
+    <!-- Breadcrumb -->
+    <xpath expr="//main/*[1]" position="before">
+        <t t-if="'has_parent_page' in main_object and main_object['has_parent_page']">
+            <div class="page_breadcrumb" data-name="Breadcrumb">
+                <t t-set="parent_page" t-value="main_object.parent_page_id"/>
+                <section>
+                    <div class="container o_not_editable py-2">
+                        <t t-if="parent_page.parent_page_id">
+                            <a t-att-href="parent_page.parent_page_id.url" t-out="parent_page.parent_page_id.name"/>
+                            <span> / </span>
+                        </t>
+                        <a t-att-href="parent_page.url" t-out="parent_page.name"/>
+                        <span> / </span>
+                        <span t-out="main_object.name"/>
+                    </div>
+                </section>
+            </div>
+        </t>
+    </xpath>
```

Comment:
- Use a bootstrap standard structure for breadcrumb
- From a SEO/standard point of view it really should be a `nav > ol > li > a` structure, not a `div > section > div > a`
- IMO we could place it *between* the `<header>` and the `<main>` instead of inside the `<main>` but I am not sure. We should:
    - Check the internet to see what's best
    - Make sure the solution works with any configuration of header and page layout in Odoo

cc @lebl-odoo

Reply:
> From a SEO/standard point of view it really should be a nav > ol > li > a structure, not a div > section > div > a

I confirm, didn't check at that time. Thx for noticing it.

> IMO we could place it between the <header> and the <main> instead of inside the <main> but I am not sure.

Both are fine. My opinion is to prefer to set it in the <main> to have a better UX experience when people use the "skip to the content" feature. Indeed, it will add context to know the hierarchy of the content, it adds value to users.

Note that a possible improvement it's to use some Schema.org attribute (inside the breadcrumb elements) for a better SEO experience. Take this on my side.